test/unit/server/get_account_test.js

```diff
@@ -71,20 +64,13 @@ describe('Server#getAccount', function () {
         jsonrpc: '2.0',
         id: 1,
         method: 'getLedgerEntries',
-        params: [
-          [
-            xdr.LedgerKey.account(
-              new xdr.LedgerKeyAccount({
-                accountId
-              })
-            ).toXDR('base64')
-          ]
-        ]
+        params: [[key.toXDR('base64')]]
       })
       .returns(
         Promise.resolve({
           data: {
             result: {
+              latestLedger: 0,
               entries: null
```

Comment:
shouldn't this now be guaranteed at least empty, `entries: []`

Reply:
yes but the RPC endpoint can still return null

Reply:
ok, cool, I was basing off the statement in pr description, may have been different context.

`entries is now guaranteed to exist, but it may be empty`

Reply:
yeah so that applies to the `Server.getLedgerEntries` _method_, while this is simulating the raw RPC response (which **will** be transformed into having `entries: []`)